### PR TITLE
Add searching to PeopleList

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13642
affects: Student
release note: Added searching to people list

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



#### Release Note: 
Added searching to people list

#### Affected Apps: Student

[MBL-13642](https://instructure.atlassian.net/browse/MBL-13642)
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/People/PeopleListViewController.swift [hunk 1]
```diff
@@ -70,6 +78,9 @@ public class PeopleListViewController: UIViewController, ColoredNavViewProtocol
 
     public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
+        if tableView.contentOffset.y == 0 {
+            tableView.contentOffset.y += searchBar.frame.height
+        }
```
Comment (armstrongnate) [inline Core/Core/People/PeopleListViewController.swift]: So easy lol.

File: Core/Core/People/PeopleListViewController.swift [hunk 2]
```diff
@@ -93,6 +110,33 @@ public class PeopleListViewController: UIViewController, ColoredNavViewProtocol
...
+
+    public func searchBarTextDidEndEditing(_ searchBar: UISearchBar) {
+        searchBar.setShowsCancelButton(false, animated: true)
+    }
+
+    public func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
+        searchBar.text = ""
+        self.searchBar(searchBar, textDidChange: "")
+        searchBar.searchTextField.resignFirstResponder()
+        tableView.setContentOffset(CGPoint(x: 0, y: searchBar.frame.height), animated: true)
```
Comment (armstrongnate) [inline Core/Core/People/PeopleListViewController.swift]: Cancel is a nice touch!

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1018280&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Pull request merged**